book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] Great job so far 👏🏻👏🏻 but I think when you want to create aretal you need a person and a book as well with a specific date so firstly in this book file you will need to pass person and date here then creating a new rental so kindly associate the book with a person when creating a rental 💯🔥✅🚀🙏🏻✨